hail/generate-build-info.sh

```diff
@@ -15,3 +16,4 @@ echo_build_properties() {
 mkdir -p src/main/resources/
 
 echo_build_properties $1 $2 > "src/main/resources/build-info.properties"
+python scripts/parse_version_info.py src/main/resources/build-info.properties python/hail/_generated_version_info.py
```

Comment:
Why not just create here python/hail/_generated_version_info.py the same way we do with build-info.properties?